src/current/v25.1/show-jobs.md

```diff
@@ -175,9 +175,10 @@ You can filter jobs by using `SHOW AUTOMATIC JOBS` as the data source for a [`SE
 
 You can display specific fields relating to [changefeed]({% link {{ page.version.version }}/create-changefeed.md %}) jobs by running `SHOW CHANGEFEED JOBS`. These fields include:
 
-* [`high_water_timestamp`]({% link {{ page.version.version }}/monitor-and-debug-changefeeds.md %}#monitor-a-changefeed): Guarantees all changes before or at this time have been emitted.
-* [`sink_uri`]({% link {{ page.version.version }}/create-changefeed.md %}#sink-uri): The destination URI of the configured sink for a changefeed.
-* `full_table_names`: The full [name resolution]({% link {{ page.version.version }}/sql-name-resolution.md %}) for a table. For example, `defaultdb.public.mytable` refers to the `defaultdb` database, the `public` schema, and the table `mytable` table.
+- [`high_water_timestamp`]({% link {{ page.version.version }}/monitor-and-debug-changefeeds.md %}#monitor-a-changefeed): Guarantees all changes before or at this time have been emitted.
+- {% include_cached new-in.html version="v25.1" %} `readable_high_water_timestamptz`: The `high_water_timestamp` value in [`TIMESTAMPTZ`]({% link {{ page.version.version }}/timestamp.md %}) format for readability.
+- [`sink_uri`]({% link {{ page.version.version }}/create-changefeed.md %}#sink-uri): The destination URI of the configured sink for a changefeed.
+- `full_table_names`: The full [name resolution]({% link {{ page.version.version }}/sql-name-resolution.md %}) for a table. For example, `defaultdb.public.mytable` refers to the `defaultdb` database, the `public` schema, and the table `mytable` table.
```

Comment:
typo: "and the table `mytable` table"

Reply:
Perfect, thank you for catching that amongst all those table tables mytable